Core/Core/Network/DownloadManager.swift

```diff
@@ -470,6 +472,60 @@ public class DownloadManager: DownloadManagerProtocol {
             debugLog("SaveFile Error", error.localizedDescription)
         }
     }
+    
+    public func removeAppSupportDirectoryDeprecatedContent() {
+        deleteMD5HashedFolders()
+    }
+    
+    private func getApplicationSupportDirectory() -> URL? {
+        let fileManager = FileManager.default
+        do {
+            let appSupportDirectory = try fileManager.url(
+                for: .applicationSupportDirectory,
+                in: .userDomainMask,
+                appropriateFor: nil,
+                create: true
+            )
+            return appSupportDirectory
+        } catch {
+            print("Error getting Application Support Directory: \(error)")
+            return nil
+        }
+    }
+    
+    private func isMD5Hash(_ folderName: String) -> Bool {
+        let md5Regex = "^[a-fA-F0-9]{32}$"
```

Comment:
Can md5 contain uppercase letters?

Reply:
True, an MD5 hash will not contain uppercase letters. However, to be safe, if there is any customization that introduces uppercase letters, it will also delete that folder.